source/sdk/threading/native/ConditionWin32.ooc

```diff
@@ -0,0 +1,87 @@
+version(windows) {
+
+	import ../Thread
+	import native/win32/[types, errors]
+	import MutexWin32
+	import structs/ArrayList
+
+	include windows
+
+	CreateEvent: extern func (...) -> Handle
+	SetEvent: extern func (Handle) -> Bool
+	CloseHandle: extern func (Handle) -> Bool
+	INFINITE: extern Long
+	WAIT_OBJECT_0: extern Long
```

Comment:
Give these ooc names:
```ooc
Infinite: extern (INFINITE) Long
WaitObject0: extern (WAIT_OBJECT_0) Long
```
Actually, you can probably think of a better name than `WaitObject0`.